dipy/data/fetcher.py

```diff
@@ -695,7 +698,8 @@ def read_cenir_multib(bvals=None):
     Notes
     -----
     Details of the acquisition and processing, and additional meta-data are
-    available through `UW researchworks <https://digital.lib.washington.edu/researchworks/handle/1773/33311>`_
+    available through `UW researchworks
+    <https://digital.lib.washington.edu/researchworks/handle/1773/33311>`_
```

Comment:
I think this would break the rendering of the URL. You can't always have <80 characters...